lib/test_suite/analyzer_verification/assert_call.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ElixirAnalyzer.TestSuite.AnalyzerVerification.AssertCall do
+  @dialyzer generated: true
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Pacman Rules
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_call "finds call to local helper function" do
+    type :informational
+    called_fn :local, &AssertCallVerification.helper/0
+    comment "didn't find a local call to helper/0"
+  end
+
+  assert_call "finds call to private local helper function" do
+    type :informational
+    called_fn :local, &AssertCallVerification.private_helper/0
+    comment "didn't find a local call to private_helper/0"
+  end
+
+  assert_call "finds call to local helper function within function" do
+    type :informational
+    calling_fn :global, &AssertCallVerification.function/0
+    called_fn :local, &AssertCallVerification.helper/0
```

Comment:
What's the difference between `:global` and `:local`? are both options valid for both `calling_fn` and `called_fn`?

Reply:
Ya, I don't know if I'm 100% happy with this part of the DSL yet. Basically I need a way to specify a the function. So if it is local, it strips off the module name to be matched. 

Maybe it would be better as a keyword list where the programmer specifies the function. I don't know. 

So while I think `:global` and `:local` would work for both directives right now, I'm not sure it makes sense for the calling_fn.

🤷‍♂️

Reply:
I've simplified this.  See new.